app/controllers/api/v2/interfaces_controller.rb

```diff
@@ -93,7 +95,7 @@ def destroy
       private
 
       def allowed_nested_id
-        %w(host_id domain_id subnet_id)
+        %w(host_id domain_id subnet_id subnet6_id)
```

Comment:
I don't think this is needed - it's for accessing items below /api/v2/subnets/:subnet_id/interfaces or similar, there won't be an subnet6_id in that instance.